cards_corrections

```diff
@@ -50,77 +30,9 @@
     "number": "48"
   },
   "Crook of Comdemnation": {
-    "name": "Crook of Condemnation",
-    "url": "http://media-dominaria.cursecdn.com/avatars/132/292/636335524953127724.png"
-  },
-  "Bontu's Last Reckoning": {
-    "url": "http://media-dominaria.cursecdn.com/avatars/132/290/636335484586372349.png"
-  },
-  "Fraying Sanity": {
-    "url": "http://media-dominaria.cursecdn.com/avatars/132/299/636335556720864056.png"
-  },
-  "Forest": {
-    "url": "http://media-dominaria.cursecdn.com/avatars/132/284/636335026018100931.png"
-  },
-  "Island": {
-    "url": "http://media-dominaria.cursecdn.com/avatars/132/281/636335023983082077.png"
-  },
-  "Mountain": {
-    "url": "http://media-dominaria.cursecdn.com/avatars/132/283/636335025652317659.png"
-  },
-  "Plains": {
-    "url": "http://media-dominaria.cursecdn.com/avatars/132/280/636335023469044373.png"
-  },
-  "Ramunap Hydra": {
-    "url": "http://media-dominaria.cursecdn.com/avatars/132/300/636335559357695252.png"
-  },
-  "Scavenger Grounds": {
-    "url": "http://media-dominaria.cursecdn.com/avatars/132/298/636335551956765662.png"
-  },
-  "Swamp": {
-    "url": "http://media-dominaria.cursecdn.com/avatars/132/282/636335025231675202.png"
-  },
-  "Swarm Intelligence": {
-    "url": "http://media-dominaria.cursecdn.com/avatars/132/302/636335674114646878.png"
-  },
-  "Wasp of the Bitter End": {
-    "url": "http://media.wizards.com/2017/hou/en_pZSTujTPgd.png"
-  },
-  "Zeakot of the God-Pharaoh": {
-    "url": "http://media.wizards.com/2017/hou/en_TogZyt1Kt9.png"
-  },
-  "Visage of Bolas": {
-    "url": "http://media.wizards.com/2017/hou/en_4uKSnFbdQL.png"
-  },
-  "Cinder Barrens": {
-    "url": "http://media.wizards.com/2017/hou/en_mTSP0FlNoq.png"
-  },
-  "Brambleweft Behemoth": {
-    "url": "http://media.wizards.com/2017/hou/en_Jxftx9lmTW.png"
-  },
-  "Avid Reclaimer": {
-    "url": "http://media.wizards.com/2017/hou/en_q2Tx8P1PsL.png"
-  },
-  "Nissa's Encouragement": {
-    "url": "http://media.wizards.com/2017/hou/en_pp5zBW3i6P.png"
-  },
-  "Woodland Stream": {
-    "url": "http://media.wizards.com/2017/hou/en_EyPJjr8lQA.png"
-  },
-  "Nicol Bolas, the Deceiver": {
-    "url": "http://media.wizards.com/2017/images/daily/en_iSHCrCgmya.png"
+    "name": "Crook of Condemnation"
   },
   "Nissa, Genesis Mage": {
-    "loyalty": "5",
-    "url": "http://media.wizards.com/2017/images/daily/en_VlI3HOtQc3.png"
-  },
-  "Reason": {
-    "url": "http://media-dominaria.cursecdn.com/avatars/132/329/636336513732500758.png"
-  },
-  "Believe": {
-    "url": "http://media-dominaria.cursecdn.com/avatars/132/329/636336513732500758.png"
```

Comment:
"Reason" is now picked up correctly - but the "Believe" part is not.

Because it's not added here? https://github.com/Cockatrice/Magic-Spoiler/blob/master/main.py#L14
But we don't do for the others either. So "Grind // Dust"" can be removed there as well?